en/drafts/originals/transcribe-handwritten-text-with-python-and-azure.md

```diff
@@ -24,132 +24,112 @@ doi: TBC
 
```

Comment:
I am also suggesting changing the lesson title to "Transcribing" rather than "Transcribe" to fit with the PH house style across our lesson directory, where we use action verbs.

Reply:
makes sense to me!